### ISSUE TITLE
Mutations for Linq methods don't work

### ISSUE THREAD

**Issue by KirillFormado** [stryker-mutator/stryker-net#579]
**Describe the bug**
This code with the Linq method doesn't mutate. 
```
public int TestLinq(int count)
{ 
    var list = Enumerable.Range(1, count);
    return list.Last();
}
```
Also I tried to move Linq method from return statement.
```
public int TestLinq(int count)
{
   var list = Enumerable.Range(1, count);
   var last = list.Last();
   return last;
}
```
And also I tried another methods (First, FirstOrDefault).
Unit test for this method.
```
[Test]
public void TestLinq_One_Equal()
{
    var r = _sl.TestLinq(5);
    Assert.AreEqual(5, r);
}
```

**Expected behavior**
Linq method should be mutated Last to LastOrDefault (etc) and vice versa.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Type of project: core
 - Framework Version: netcoreapp2.2
 - Stryker Version: 0.11.0 (beta)

**Additional context**
Others mutation for another methods works well.

**Comment by richardwerkman** [comment]
Some extra context: this did work prior to version 0.9. My guess is that this has to do with the refactoring of the Mutant Orchestrator. Our tests did not pick this up as we only unit test the linq mutator and not if it is called as expected. No linq mutations are present in the integration test either.

**Comment by richardwerkman** [comment]
I have just figured out why the LinqMutator is not called. [Here](https://github.com/stryker-mutator/stryker-net/blob/bb5b7dc0bc7c61f7692339261c348347899323ef/src/Stryker.Core/Stryker.Core/Mutators/LinqMutator.cs#L46) you can see the mutator targets `MemberAccessExpressionSyntax`. [Here](https://github.com/stryker-mutator/stryker-net/blob/bb5b7dc0bc7c61f7692339261c348347899323ef/src/Stryker.Core/Stryker.Core/Mutants/MutantOrchestrator.cs#L248) you can see the `MutantOrchestrator` skips this type of node... Thus the mutator is never called. 

So to fix this issue we should: 
 - Make sure the mutant orchestrator mutates the node type `MemberAccessExpressionSyntax`.
 - Or refactor the LinqMutator to target some other `ExpressionSyntax`.

My preference goes to fixing the `MutantOrchestrator` as the `LinqMutator` works fine as it is now. By specifically targeting only `MemberAccessExpressionSyntax` the LinqMutator doesn't get called too many times. If we would target `ExpressionSyntax` it would get called much more during the mutation phase.

**Comment by KirillFormado** [comment]
I've tried to fix this bug, but encountered this issue with code mutation. After the fix, LINQ methods have started mutate but they mutate in not compilable code like this 
`StrykerVroQybzvjujF1o7.ActiveMutationHelper.ActiveMutation==0?stringList.FirstOrDefault:stringList.First)()`
This caused the compile error "CS0173 Type of conditional expression cannot be determined because there is no implicit conversion between 'method group' and 'method group'".
I thought about fixing this like changing ternary expression to return result of the method instead of method group, but I'm not familiar with Roslyn API, so in progress now.
I curios, how it works before. So need to investigate it too.

**Comment by richardwerkman** [comment]
I have to say that the `MutantOrchestator` is one of the most complex parts of stryker so I think this might be a hard bug to fix. I've created a branch where I've cleaned the mutant orchestrator tests (579_call_linq_mutator). This makes TDD possible to target this bug. Maybe that could help. I will try to tackle this too if I find time soon.

**Comment by dupdob** [comment]
Fixed by PR #624
